# user_request.srv
int32 requested_value
---
bool success
geometry_msgs/Point position

